com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -242,7 +242,11 @@ public int getMaxCpuThreads() {
         if (maxThreadsOpt == null) {
             return getDefaultMaxCpuThreads();
         }
-        return Integer.parseInt(maxThreadsOpt);
+        int threads = Integer.parseInt(maxThreadsOpt);
+        if (threads == 0) {
+            threads = Runtime.getRuntime().availableProcessors();
+        }
+        return threads;
     }
```

Comment:
Could you please also add log message with something like `("Build uses %s threads", getMaxCpuThreads())` - it will make CI configuration much easier

Reply:
There is logging in TaskBuilder.java and ArchiveBuilder.java when the thread pools are created. That should be enough?